doc/cookbook/custom_dna4.cpp

```diff
@@ -0,0 +1,58 @@
+#include <seqan3/alphabet/nucleotide/nucleotide_base.hpp>
+#include <seqan3/core/debug_stream.hpp>
+
+class my_dna4 : public seqan3::nucleotide_base<my_dna4, 4>
+{
+public:
+    using nucleotide_base<my_dna4, 4>::nucleotide_base; // Pull constructors of the base class into public scope.
```

Comment:
That is a wrong sentence because you cannot change the access modifiers of constructors through a using expression.